app/views/categories/_nav.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="d-flex justify-content-between" style="margin-bottom: 10px">
```

Comment:
- [ ] You should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

Colors
Typography: font face, size and weight.
Layout: composition and space between elements.

### Your Implementation
![image](https://github.com/mahdinoori2000/budget-app/assets/45460192/b538a7f3-e24b-44fe-9235-9bde3ccf8489)

### Required
![image](https://github.com/mahdinoori2000/budget-app/assets/45460192/40ece178-affe-429f-aede-b6e50bde556f)

Your UI should look very similar to that provided in the original for example, the button should:
- at the bottom of the page
- color #5fb523 with a white border surrounding the text

The layout of the cards should be the same in the original design with the image covering the left bit.

Like I mentioned, make your UI very similar to the original design. Compare your UI with that provided in the project requirements and make sure they look similar .